projects/README.md

```diff
@@ -41,7 +41,7 @@
 |[RapidXml](http://rapidxml.sourceforge.net/ 'RapidXml on sourceforge')|[open](http://rapidxml.sourceforge.net/license.txt 'Boost Software License -or- The MIT License')|fast XML parser|1.13|[repo](https://github.com/smanders/rapidxml 'rapidxml repo on github')|[diff](https://github.com/smanders/rapidxml/compare/v1.13...xp1.13 'patch/diff')|
 |[shapelib](http://shapelib.maptools.org/ 'Shapefile C Library website')|[open](http://shapelib.maptools.org/license.html 'MIT Style -or- LGPL')|reading, writing, updating ESRI Shapefiles|1.2.10|[repo](https://github.com/smanders/shapelib 'forked shapelib repo on github')|[diff](https://github.com/smanders/shapelib/compare/v1.2.10...xp1.2.10 'patch/diff')|
 |[wirehair](https://github.com/catid/wirehair 'wirehair repo on github')|[open](https://github.com/catid/wirehair/blob/master/LICENSE.txt 'BSD 3-Clause New or Revised License')|fast and portable fountain codes in C|20.11.21|[repo](https://github.com/catid/wirehair 'wirehair repo on github')|[diff](https://github.com/smanders/wirehair/compare/catid:9e8923a200045c7bdb9e89b449692dbfed6b6de0...xp20.11.21 'patch/diff')|
-|[wxWidgets](http://wxwidgets.org/ 'wxWidgets website')|[open](http://www.wxwidgets.org/about/newlicen.htm 'wxWindows License: essentially LGPL with an exception')|Cross-Platform GUI Library|3.1.0|[repo](https://github.com/wxWidgets/wxWidgets 'wxWidgets repo on github')|[diff](https://github.com/smanders/wxWidgets/compare/v3.1.0_pkg...xp3.1.0 'patch/diff')|
```

Comment:
I figured after https://github.com/smanders/wxcmake/pull/1 and https://github.com/smanders/wxWidgets/pull/10 get merged we could change this back to smanders. Either I can do that or you, its all the same to me.